docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Run the container the default way
+if [ "$1" = 'timesketch' ]; then
+	# Set SECRET_KEY in /etc/timesketch.conf if it isn't already set
+	if grep -q "SECRET_KEY = u''" /etc/timesketch.conf; then
+		OPENSSL_RAND=$( openssl rand -base64 32 )
+		# Using the pound sign as a delimiter to avoid problems with / being output from openssl
+		sed -i 's#SECRET_KEY = u\x27\x27#SECRET_KEY = u\x27'$OPENSSL_RAND'\x27#' /etc/timesketch.conf
+	fi
+
+	# Set up the Postgres connection
+	if [ $POSTGRES_USER ] && [ $POSTGRES_PASSWORD ] && [ $POSTGRES_ADDRESS ] && [ $POSTGRES_PORT ]; then
```

Comment:
It would be nice to change the postgresql password here. Is that possible? I'm afraid that we will end up with people having exposed postgresql servers.